roles/matrix-synapse/templates/synapse/homeserver.yaml.j2

```diff
@@ -302,7 +302,7 @@ listeners:
 {% if matrix_synapse_federation_port_enabled %}
   # Unsecure HTTP listener (Federation API): for when matrix traffic passes through a reverse proxy
   # that unwraps TLS.
-  - port: 8048
+  - port: {{ matrix_synapse_container_default_federation_port }}
```

Comment:
`{{ matrix_synapse_container_default_federation_port|to_json }}` would be better.

Reply:
I agree